### DISCUSSION TITLE
TTF and WOFF2 lacks support for Æ and Ø in different weigths

### DISCUSSION THREAD

**Discussion by simenwik** [ivodolenc/aspekta#21] [Ideas]
### Steps to Reproduce

Install font files locally or on web
Use fonts and enter the characters Æ and Ø

### Current Behavior

Æ and Ø does not match the selected weights. 

### Expected Behavior

Æ and Ø should match the selected weights

### Additional Details

I have tested the TTF variable, TTF standard and WOFF2 standard font files. For the variable TTF the danish and norwegian characters Æ and Ø only matches regular weight (400). When using the standard TTF and WOFF2 files Æ and Ø matches regular and 700, 750 or 800 (I'm not certain witch). 

![image](https://github.com/user-attachments/assets/ce5fc528-6f40-4c7c-81f7-686973ae181f)

### Version

chore(release): bump version to 2.000

**Comment by ivodolenc** [comment]
Hi, these chars `Æ`, `æ`, `Ø (Oslash)` and `ø (oslash)` are not yet implemented so they can't be displayed correctly. To be clear, it's not a bug.

Since I'm not too familiar with these chars and langs, I should do some additional research to see exactly where they belong and what other glyphs need to be added to support additional languages ​​like `Danish`, `Norwegian`, etc.

It would be great if you could put here a list of all chars i.e. glyphs missing to fully support languages ​​like `Danish` and `Norwegian`, it would make the research a lot easier.

As soon as I have more time I will come back to this. The plan is to support as many languages ​​as possible to make the font better and more accessible to a wider audience. Anyway, thanks for reporting 👍🏻

Also, I'm open for PRs, if anyone is interested in helping out.

**Reply by simenwik** [reply]
Thanks for the quick reply. The only characters missing from the Norwegian and Danish alphabet is Æ and Ø, so if you add those the font should have full support for Norwegian and Danish language.

**Comment by ivodolenc** [comment]
Just to note, this was added in the latest version `2.100` ( #26 ), so feel free to try it out 👍🏻
